frontend/src/libs/components/profile-settings-sidebar/profile-settings-button/styles.module.scss

```diff
@@ -29,19 +40,16 @@
 
 .text-container {
   display: flex;
+  flex: 1;
   align-items: center;
 }
 
-.icon-background {
+.arrow-container {
   display: flex;
+  flex-direction: column;
   justify-content: center;
   align-items: center;
-  width: 83px;
-  height: 83px;
-  margin-right: 25px;
-  padding: 3px;
-  background-color: var(--blue-300);
-  border-radius: 20px;
+  margin-left: 26px;
 }
 
 @media (width <= 480px) {
```

Comment:
It should be (width < 480px), but I guess its one of the parts that you pulled from development